### DISCUSSION TITLE
File Manager Extension installation?

### DISCUSSION THREAD

**Discussion by Perkolator** [tristanheaven/gtkhash#128] [Q&A]
The installation instructions for File Manager Extensions is rather lacking. Is the distro repo the only choice to install the extensions?

Just tried to install the main app as Flatpak and then only the extension from the distro repo. The result is that apt wants to also install the old version of gtkhash from the repo.

Mint 20 has both the main app and the extension as 1.2 versions, almost 3 years old.

**Comment by tristanheaven** [comment]
It's unfortunate the Debian package hasn't been updated as that was the best way to install the extensions.

**Reply by Perkolator** [reply]
The only way?
